packages/experimental/Spinner/src/Spinner.styling.win32.ts

```diff
@@ -0,0 +1,62 @@
+import type { UseStylingOptions, Variant } from '@fluentui-react-native/framework';
+import { buildProps } from '@fluentui-react-native/framework';
+
+import type { SpinnerProps, SpinnerSlotProps, SpinnerTokens } from './Spinner.types.win32';
+import { spinnerName } from './Spinner.types.win32';
+import { defaultSpinnerTokens } from './SpinnerTokens';
+
+export const diameterSizeMap: { [key: string]: number } = {
```

Comment:
As noted in the comment on SpinnerTokens, this would usually be done in the tokens file and styling would map tokens to style prop entries.